dist/pythonlibs/riotctrl_shell/gnrc.py

```diff
@@ -0,0 +1,189 @@
+# Copyright (C) 2019-20 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+"""
+Mixins for GNRC-related shell interaction
+
+Defines mixins for GNRC-related shell commands
```

Comment:
AFAICS, there are no more mixins in this module

Reply:
You can still use them as mixins, but will remove.